src/libs/errors.ts

```diff
@@ -0,0 +1,5 @@
+export class InputError extends Error {
```

Comment:
Is this error for Input from the User or Input in general?

Reply:
It's for any input that is received as part of a request, basically anything that we should raise a 400 error on.

Reply:
for both of these, adding a comment to specify that would be helpful in my opinion.